test/integration/change-streams/change_stream.test.ts

```diff
@@ -1816,6 +1829,78 @@ describe('Change Streams', function () {
       });
     });
   });
+
+  describe("NODE-4763 - doesn't produce duplicates after resume", function () {
+    const resumableError = { code: 6, message: 'host unreachable' };
+
+    beforeEach(async function () {
+      await client.db('admin').command({
+        configureFailPoint: is4_2Server(this.configuration.version)
+          ? 'failCommand'
+          : 'failGetMoreAfterCursorCheckout',
+        mode: { skip: 1 },
+        data: {
+          failCommands: ['getMore'],
+          errorCode: resumableError.code,
+          errmsg: resumableError.message
+        }
+      } as FailPoint);
+    });
+
+    afterEach(async function () {
+      await client.db('admin').command({
+        configureFailPoint: is4_2Server(this.configuration.version)
+          ? 'failCommand'
+          : 'failGetMoreAfterCursorCheckout',
+        mode: 'off'
+      } as FailPoint);
+    });
+
+    describe('when using iterator form', function () {
+      it('#next', { requires: { topology: 'replicaset' } }, async function test() {
+        await initIteratorMode(changeStream);
+
+        await collection.insertOne({ a: 1 });
+        const change = await changeStream.next();
+        expect(change).to.have.property('operationType', 'insert');
+        expect(change).to.have.nested.property('fullDocument.a', 1);
+
```

Comment:
[optional, applies to all similar assertions in the PR]


```suggestion
        const change = await changeStream.next();
        expect(change).to.containSubset({
          operationType: 'insert',
          fullDocument: {
            a: 1
          }
        });
```

Reply:
Thank you! Applied to all newly added tests.